notebooks/3_KernelMethods.ipynb

```diff
@@ -35,9 +35,12 @@
     "from utilities.plotting import (\n",
     "    plot_projection, plot_regression, check_mirrors, get_cmaps, table_from_dict\n",
     ")\n",
-    "from utilities.kernels import linear_kernel, gaussian_kernel, center_kernel\n",
-    "from utilities.classes import PCA, LR, KPCA, KRR, KPCovR\n",
-    "\n",
+    "from utilities.kernels import linear_kernel, gaussian_kernel,center_kernel # to be replaced with functions from skcosmo\n",
```

Comment:
if we don't use the kernel-tutorials kernels, please delete this line.

Reply:
Okay.